src/cuda/cuda-constant-buffer-pool.cpp

```diff
@@ -65,8 +52,8 @@ Result ConstantBufferPool::allocate(size_t size, Allocation& outAllocation)
         Page& page = m_largePages.back();
         SLANG_RETURN_ON_FAIL(createPage(size, page));
         page.usedSize = size;
-        outAllocation.hostData = page.hostData;
-        outAllocation.deviceData = page.deviceData;
+        outAllocation.hostData = page.handle->getHostPtr();
```

Comment:
Worth thinking about keeping `hostData` and `deviceData` members in `Page`. Makes debugging easier and less indirection in debug builds. But fine to keep as is (not having redundant data is probably better).

Reply:
agreed